yarn.lock

```diff
@@ -2834,11 +2834,121 @@
     url-loader "^4.1.1"
     webpack "^5.73.0"
 
+"@esbuild/android-arm64@0.19.8":
```

Comment:
You've added esbuild `0.19.8`, but the project workspace uses `0.14.54`, hence the new dependencies in this file. Please upgrade the rest of the project so we're removing this dupe - esbuild is a binary platform-specific blob we need to download and has a non-trivial impact on the dev dependency footprint.